### ISSUE TITLE
Massive fps drops, it's only fine in cars 

### ISSUE THREAD

**Issue by harm-less** [LukeRoss00/gta5-real-mod#110]
Hello Luke,

Thanks for the amazing work you've done! I once tried a VorpX mod, but that made me really sick, and years later I tried again and it's pure joy 😄 

However, I don't know why, but most of the times I I'm NOT in a car, the framerate just drop to 40 while in a car it stays between 100 and 120 in a car. The GPU (RTX 2080) is also working at about 30%.

Once I get out, it drops, once I get back into the car, smooth sailing.
I'm using the Manual Transmission mod for my wheel, perhaps that has something to do with it. I'll try at least try this myself.

Any idea what this could be as well?

Thanks,
Harm

**Comment by LukeRoss00** [comment]
Hey Harm! I've never heard of this particular issue. I don't think it has to do with my mod, because nobody else has observed a similar behavior that I know of. Many other people are also using the Manual Transmission mod with success, so in your case it might be a configuration problem?

Do you have correct FOV both in car and on foot?

**Comment by harm-less** [comment]
I do have similar FOV in both.

The manual transmission mod doesn't seem to be the problem as well.

I did discover it corrected itself after switching to another window, but I'm not sure if that was just a fluke or a "fix".

**Comment by harm-less** [comment]
I took a gamble and upgraded my HDD to a M.2, man, night and day in terms of performance. I guess in my case at least the amount of files needed from disk was holding back the rest of the system.

I first noticed it when I was downloading a game from Steam whilst simultaneously playing a demanding, regular VR game a while back. This stuck with me and I decided to buy a M.2 this weekend (I guess an SSD would have been sufficient as well) and moved the game there.

So yeah, don't know what else to say, but I can run the game on medium settings and MSAA 2x + TXAA turned on and it's just awesome! Thanks a bunch, I'll definitely enjoy your mod for a lot more hours!

I hope this this issue can help someone else as well.
